recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -105,6 +105,9 @@ do_deploy() {
     if [ -n "${HDMI_MODE}" ]; then
         sed -i '/#hdmi_mode=/ c\hdmi_mode=${HDMI_MODE}' $CONFIG
     fi
+    if [ -n "${HDMI_CVT}" ]; then
```

Comment:
Could you also add a documentation commit for this? Thanks, Otavio.

Reply:
Where?

Reply:
https://github.com/agherzan/meta-raspberrypi/blob/master/docs/extra-build-config.md would be great.

Reply:
done.

Reply:
That will do. Thanks.